modules/oe_translation_cdt/xml/schema.xsd

```diff
@@ -0,0 +1,61 @@
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" attributeFormDefault="unqualified" elementFormDefault="qualified">
```

Comment:
I've put this schema in the module to have a clear image of the XML format for later.